.github/workflows/ubuntu_build.yml

```
@@ -13,9 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: sudo apt-get -q update
-      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq upgrade
-      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install
-             libcurl4-openssl-dev libssl-dev python3-dev python3-pip
+      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq --no-install-recommends dist-upgrade
+      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq --no-install-recommends install ca-certificates curl python3-dev libcurl4-openssl-dev gcc libssl-dev
```

void right-left scrolling, improve readability and sort dependancies to avoid duplicates.
```suggestion
      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq --no-install-recommends install ca-certificates
                 curl gcc libcurl4-openssl-dev libssl-dev python3-dev 
```